### PR TITLE
feat: change copies from input field in Offer

### PR DESCRIPTION
**Pull request by araujobarret**
The type of this PR is: **Feature**

This PR solves [NX-3111]

### Description

Changes the copies from the low amount warning in the Offer. Slightly adjust the design removing underline decoration and not breaking the line anymore/

Price
Screenshot 2022-03-23 at 10 40 09

Price Range
Screenshot 2022-03-23 at 10 51 51

@artsy/negotiate-devs 

[NX-3111]: https://artsyproduct.atlassian.net/browse/NX-3111?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/v2/Apps/Order/Components/MinPriceWarning.tsx [hunk 1]
```diff
@@ -22,19 +22,15 @@ export const MinPriceWarning: React.FC<MinPriceWarningProps> = ({
       flow: AnalyticsSchema.Flow.MakeOffer,
       order_id: orderID,
     })
-  }, [])
+  }, [orderID, tracking])
```
Comment (araujobarret) [inline src/v2/Apps/Order/Components/MinPriceWarning.tsx:25]: nit: added missing deps from the hook, checked and it's not triggered more than once

File: src/v2/Apps/Order/Components/MinPriceWarning.tsx [hunk 2]
```diff
@@ -22,19 +22,15 @@ export const MinPriceWarning: React.FC<MinPriceWarningProps> = ({
...
 
   return (
     <Clickable cursor="pointer" onClick={onClick} mt={2}>
       <Message variant="warning" p={2}>
         <Text>
-          Galleries usually accept offers within
           {isPriceRange
```
Comment (lean8086) [inline src/v2/Apps/Order/Components/MinPriceWarning.tsx:31]: Strange that we don't test both of these use cases.

**Review by lean8086 — DISMISSED** [review]
LGTM!

File: src/v2/Apps/Order/Components/MinPriceWarning.tsx (hunk 2 above)
Comment (araujobarret) [reply src/v2/Apps/Order/Components/MinPriceWarning.tsx:31]: Actually, they were in a different component, fixed and improved in this [commit](https://github.com/artsy/force/pull/9714/commits/f5053a9051b2afb65493b665acdef18564cde3ca)

**Pull request merged**